example/babel.config.js

```diff
@@ -1,13 +0,0 @@
-module.exports = function (api) {
```

Comment:
Does the normal example still work after you deleted all of this?

Reply:
Yep, you just need to do `npm run start` and then `npm run start:ios` or `npm run start:android`. All the dependencies are now pointing to the root `node_modules`.